src/components/NavBar/index.jsx

```diff
@@ -52,6 +52,9 @@ const NavBar = ({ isTesting }) => {
               <Icon style={{ transform: "scale(2)" }} component={AdminSvg} />
             </Link>
           </Menu.Item>
+          <Menu.Item key="6" className="navbarLogin">
+            <Link to="/register">Sign-In/Sign-Up</Link>
```

Comment:
why is there 1 link that points to signin and signup page they should be seprate

Reply:
Beacause according to design, the sign in and sign out in same page and it's combined with a button.